NOTICE

```diff
@@ -0,0 +1,7 @@
+Jaeger, Distributed Tracing Platform.
+
+Copyright 2015-2019 The Jaeger Project Authors
+
+Licensed under Apache License 2.0.  See LICENSE for terms.
+
+Includes software developed at Uber Technologies, Inc. (https://eng.uber.com/).
```

Comment:
@caniszczyk WDYT here? When we transferred Envoy we relicensed/copyrighted everything. I just want to make sure we won't run into any issues in the future.